app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  EMAIL_FORMAT = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
```

Comment:
protip: email is really common regex - you can use custom validators flow here https://guides.rubyonrails.org/active_record_validations.html#custom-validators